examples/cmri_mpca/main.py

```diff
@@ -0,0 +1,86 @@
+"""
+PyKale Tutorial: A Machine Learning Pipeline for PAH Diagnosis
+
+Reference:
+Swift, A. J., Lu, H., Uthoff, J., Garg, P., Cogliano, M., Taylor, J., ... & Kiely, D. G. (2020). A machine learning
+cardiac magnetic resonance approach to extract disease features and automate pulmonary arterial hypertension diagnosis.
+European Heart Journal-Cardiovascular Imaging. https://academic.oup.com/ehjcimaging/article/22/2/236/5717931
+"""
+import os
+
+import numpy as np
+import pandas as pd
+from config import get_cfg_defaults
+from sklearn.model_selection import cross_validate
+
+from kale.interpret import model_weights, visualize
+from kale.loaddata.get_dicom import read_dicom_images
+from kale.pipeline.mpca_trainer import MPCATrainer
+from kale.prepdata.image_transform import mask_img_stack, normalize_img_stack, reg_img_stack, rescale_img_stack
+from kale.utils.download import download_file_by_url
+
+
+def main():
+    # ---- setup configs ----
+    cfg_path = "configs/tutorial_svc.yaml"  # Path to `.yaml` config file
```

Comment:
Take a look at other examples please. This hard-coded config path is inconsistent with the readme standard command line input.